a10_octavia/controller/worker/flows/a10_listener_flows.py

```diff
@@ -73,7 +62,7 @@ def get_create_all_listeners_flow(self):
         create_all_listeners_flow.add(database_tasks.ReloadLoadBalancer(
             requires=constants.LOADBALANCER_ID,
             provides=constants.LOADBALANCER))
-        create_all_listeners_flow.add(amphora_driver_tasks.ListenersUpdate(
+        create_all_listeners_flow.add(virtual_port_tasks.ListenersUpdate(
```

Comment:
The "get_create_all_listeners_flow" is not being used anywhere, need to remove this flow or start support for this flow. 
Also ListenersUpdate takes 3 args,

Reply:
Going to leave it in for this PR since it's targeted towards styling fixes. We'll remove this during listener flow refactor